### PR TITLE
implemented word association function

### PR DESCRIPTION
**Pull request by benzeg**
- removed home page router
- removed api key
- added server start script

### CODE REVIEW

File: server/lib/request-handler.js [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+
+  //req.body.term comes in as an array of search terms input form user
+  //convert search term array to string
+  var searchTerms = req.body['term'].slice();
+  searchTerms.toString();
+  searchTerm.concat(searchTerms);
+
+  //sends post request to Twinword API with user entered search term
+  unirest.post("https://twinword-word-associations-v1.p.mashape.com/associations/")
+    .header("X-Mashape-Key", "uSuQU8KYwRmshWTMvvi8MXcuEgn2p1IeI8BjsnPoS7M2U4ujCs")
```
Comment (JamesFalkoff) [inline server/lib/request-handler.js]: This looks like an API key -- we should probably not put API keys up on Github.

File: server/server-config.js [hunk 2]
```diff
@@ -11,6 +11,6 @@ app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname + '/../client')));
 
 app.get('/', handler.renderIndex);
```
Comment (sscchan) [inline server/server-config.js]: If we're using express.static, we shouldn't need to manually handle the '/' endpoint for serving static pages.

File: server/lib/request-handler.js [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
+      res.writeHead(200).send(result.body['associations_array']);
+  });
 }
 
 //Renders home page
 exports.renderIndex = function(req, res) {
-	res.sendFile(path.join(__dirname, '/../../client/index.html'));
-}
\ No newline at end of file
+  res.sendFile(path.join(__dirname, '/../../client/index.html'));
```
Comment (sscchan) [inline server/lib/request-handler.js]: If we're using express.static, we shouldn't need to manually handle the '/' endpoint for serving static pages. And we don't need the renderIndex method.

File: server/lib/request-handler.js (hunk 1 above)
Comment (sscchan) [reply server/lib/request-handler.js]: We might need to regen this key now that it's in the open....

File: package.json [hunk 4]
```diff
@@ -31,6 +31,7 @@
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "request": "^2.79.0",
+    "unirest": "^0.5.1",
     "webpack": "^1.14.0",
     "webpack-dev-server": "^1.16.2"
   },
```
Comment (sscchan) [inline package.json:38]: Should we add a script to run the sever in package.json (now that the server is functional)?

**Review by sscchan — COMMENTED** [review]
There's some duplication/redundancy with handling '/' endpoint and using express.static middleware.

File: package.json (hunk 4 above)
Comment (benzeg) [reply package.json:38]: Added script to start nodemon on npm start

**Pull request merged**